cpp/src/io/utilities/hostdevice_vector.hpp

```diff
@@ -173,7 +172,7 @@ class hostdevice_vector {
   }
 
  private:
-  cudf::detail::rmm_host_vector<T> h_data;
+  cudf::detail::host_vector<T> h_data;
```

Comment:
Question: Now we have unified memory, why can't we just use a unified-memory-allocated array instead of two separate host+device arrays like this? Lower speed?

Reply:
the performance was significantly worse with direct kernel access to pinned memory, so we make these explicit copies.